WordPress/gutenberg
# Site Editor: Add the Post Author component to the Page sidebar

Related #52632 

## What?

This PR adds the "post author" control from the post editor to the site editor sidebar. The goal is to ultimately bridge the gap between these two editors and just merge them.

## Testing Instructions

1- Open a page in the site editor.
2- Notice that you can change the author from the sidebar.
